### ISSUE TITLE
Text spacing content overlaps

### ISSUE THREAD

**Issue by schne324** [dequelabs/cauldron#569]
<dl><dt>Description</dt><dd>When text spacing is adjusted, some content is unclear because it overlaps with other content.
Text spacing content overlaps. 
Elements - 
1. I'm a really big button with lots of text pointout
2. Change my position pointout</dd><dt>Impact</dt><dd>Serious</dd><dt>Recommendation</dt><dd>RULE : 
The spacing between letters, words, lines of text and paragraphs MUST be adjustable.

HOW TO FIX: 
Fix this issue by allowing users to override text spacing styles by providing flexibility in the height and width of content containers. Some common techniques include:
1. Size containers to a have a value greater than the default width of the text. Small areas of text (such as navigation bars) need to have sufficient space within each box so that words and characters do not expand out of their containers.
2. Use the inline-block value of the CSS display property and do not set negative margins to help allow for spacing overrides. 
 3. Do not set the CSS height property, it can help ensure paragraphs are able to expand.

REFERENCE: 
WCAG Understanding document: https://www.w3.org/WAI/WCAG21/Understanding/text-spacing

BACKGROUND: 
Some people with vision, reading, and cognitive disabilities have trouble reading particular fonts or font weights, or need more space between letters, words, or paragraphs to effectively read text content. Text content and containers should be styled so that they give the user the flexibility to adjust line spacing (to at least 1.5 times the font size), paragraph spacing (to at least 2 times the font size), letter spacing (to at least 0.12 times the font size) and word spacing (to at least 0.16 times the font size) without truncating or overlapping content. Allowing for flexibility in text spacing allows users to apply their own stylesheet, bookmarklet, extension, or application to adjust spacing and/or fonts to meet their personal needs.</dd><dt>Source code</dt><dd><pre><code><button class="Button--primary" style="margin-bottom: 171px;">I'm a really big button with lots of text</button></code></pre></dd><dt>Checkpoint</dt><dd>Text Spacing (1.4.12.a)</dd><dt>URL</dt><dd>https://cauldron.dequelabs.com/components/Pointout</dd></dl>

**Comment by schne324** [comment]
`<Pointout />` components are supposed to overlay content (and are given `role=region`) - this is not an issue
